src/name/dns_name.rs

```diff
@@ -139,6 +139,7 @@ impl core::fmt::Debug for DnsNameRef<'_> {
     }
 }
 
+/// Requires the `alloc` feature.
```

Comment:
Unintentional? But either way this is a fallback if the alloc feature is unavailable.

Reply:
Yes, copy & paste shenanigan.